pkg/controllers/node/termination/suite_test.go

```diff
@@ -341,15 +342,18 @@ var _ = Describe("Termination", func() {
 			Expect(env.Client.Delete(ctx, node)).To(Succeed())
 			node = ExpectNodeExists(ctx, env.Client, node.Name)
 			ExpectObjectReconciled(ctx, env.Client, terminationController, node)
-			ExpectSingletonReconciled(ctx, queue)
 
 			// Expect node to exist and be draining
 			ExpectNodeWithNodeClaimDraining(env.Client, node.Name)
 
+			// Expect podNoEvict to be added to the queue
+			Expect(queue.Has(podNoEvict)).To(BeTrue())
+
+			// Attempt to evict the pod, but fail to do so
+			ExpectSingletonReconciled(ctx, queue)
+
 			// Expect podNoEvict to fail eviction due to PDB, and be retried
-			Eventually(func() int {
-				return queue.NumRequeues(terminator.NewQueueKey(podNoEvict))
```

Comment:
is there no more numRequeues in the new interface?

Reply:
Correct. The current interface always returns 0. The updated testing does the same check though.